lms/djangoapps/student_account/context_processor.py

```diff
@@ -0,0 +1,33 @@
+# from common.djangoapps.third_party_auth.pipeline import get_provider_user_states
```

Comment:
please remove this line if it is no longer needed, thanks

Reply:
remove